.github/workflows/auto_label_pr.yml

```diff
@@ -0,0 +1,22 @@
+# inspired by polkadot
```

Comment:
Just a question:
This means if draft is set for new PR, no S1-pleasereview and if draft is not set for new PR, the label S1-pleasereview will be tagged?